.github/workflows/check_readme_cli.yaml

```
@@ -0,0 +1,67 @@
+name: Check README CLI Help
+
+on:
+  pull_request:
+    paths:
+      - 'cmd/ethrex/cli.rs'
+      - 'cmd/ethrex/Cargo.toml'
```

If we change the `cmd/ethrex/l2.rs` file and the `l2` feature is active, we will get another `help` message. Isn't it better to have the following (and also run the `--help` command with the l2 feature active)?:
```suggestion
      - 'cmd/ethrex/**'
```
Or maybe we can have a different job for the L2's docs.

---

The l2 cli is not currently in the README. But it could be added and this job adapted in a follow up.

Anyway, I did apply your suggestion in 3ba82aa830742c08e446f36a672710e9ee686a9b